src/libraries/Common/src/System/Net/SocketProtocolSupportPal.Windows.cs

```diff
@@ -76,14 +39,14 @@ private static bool IsProtocolSupported(AddressFamily af)
             }
             finally
             {
-                if (s != IntPtr.Zero)
+                if (s != INVALID_SOCKET)
```

Comment:
What in the above try block can throw? If it's Interop.Winsock.WSASocketW, `s` could still be null.

Reply:
It shouldn't other than something catastrophic (like the Win32 method not existing).  But I fixed it.